document/tsrc/swreg.tex

```diff
@@ -0,0 +1,6 @@
+\begin{longtable}{|l|c|c|r|P{0.8cm}|p{5.5cm}|}
```

Comment:
This file seems to be  not necessary, and the template in iob-lib is being used correctly in this case, where all swregs are grouped in the same table

Only if we want something different must we add this file